libpod/pod_top_linux.go

```diff
@@ -0,0 +1,51 @@
+// +build linux
+
+package libpod
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/containers/psgo"
+)
+
+// GetPodPidInformation returns process-related data of all processes in
+// the pod.  The output data can be controlled via the `descriptors`
+// argument which expects format descriptors and supports all AIXformat
+// descriptors of ps (1) plus some additional ones to for instance inspect the
+// set of effective capabilities.  Eeach element in the returned string slice
+// is a tab-separated string.
+//
+// For more details, please refer to github.com/containers/psgo.
+func (p *Pod) GetPodPidInformation(descriptors []string) ([]string, error) {
```

Comment:
Should probably lock the pod during this function, to ensure that it isn't removed out from under you

Reply:
fixed!